<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(148, 237, 253, 255), stop:0.142045 rgba(166, 241, 253, 255), stop:0.346369 rgba(177, 200, 250, 255), stop:0.517045 rgba(218, 170, 244, 255), stop:0.793041 rgba(249, 132, 246, 255), stop:0.991905 rgba(246, 142, 228, 255));
font-family: Disco Duck;
cursor: Hotline Black Normal Select</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="player_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Player&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="music_start">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	color: rgb(22, 158, 67);
background-color: rgba(255,255,255,5);
border: 1px solid rgba(255,255,255,5);
border-radius: 7px;
width: 230px;
height: 50px;
font:30pt;
}
QPushButton:hover{
background-color: rgba(255, 225, 225,80);
font:25pt;
}
QPushButton:pressed{
background-color: rgba(255,255,255,150);
font:24pt;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icon/icon/play.png</normaloff>:/icon/icon/play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="music_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
font: 16pt;</string>
    </property>
    <property name="text">
     <string>Music</string>
    </property>
   </widget>
   <widget class="QLabel" name="computer_title">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>129</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Computer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>710</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="rock_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgba(255,255,255,2);
border: 1px solid rgba(255,255,255,2);
border-radius: 7px;
width: 230px;
height: 50px;
font:16pt;
}
QPushButton:hover{
background-color: rgba(255,255,255,50);
font:15pt;
}
QPushButton:pressed{
background-color: rgba(255,255,255,90);
font:14pt;
}
</string>
       </property>
       <property name="text">
        <string>Rock</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scissors_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgba(255,255,255,2);
border: 1px solid rgba(255,255,255,2);
border-radius: 7px;
width: 230px;
height: 50px;
font:16pt;
}
QPushButton:hover{
background-color: rgba(255,255,255,50);
font:15pt;
}
QPushButton:pressed{
background-color: rgba(255,255,255,90);
font:14pt;
}
</string>
       </property>
       <property name="text">
        <string>Scissors</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="paper_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgba(255,255,255,2);
border: 1px solid rgba(255,255,255,2);
border-radius: 7px;
width: 230px;
height: 50px;
font:16pt;
}
QPushButton:hover{
background-color: rgba(255,255,255,50);
font:15pt;
}
QPushButton:pressed{
background-color: rgba(255,255,255,90);
font:14pt;
}
</string>
       </property>
       <property name="text">
        <string>Paper</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="back_to_menu">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	
background-color: rgba(255,255,255,5);
border: 1px solid rgba(255,255,255,5);
border-radius: 7px;
width: 230px;
height: 50px;
font:16pt;
}
QPushButton:hover{
background-color: rgba(255, 225, 225,50);
font:15pt;
}
QPushButton:pressed{
background-color: rgba(255,255,255,90);
font:13pt;
}
</string>
    </property>
    <property name="text">
     <string>Back to
 the menu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
  <include location="cursor.qrc"/>
 </resources>
 <connections/>
</ui>
